python/modules/IcePy/Config.h

```diff
@@ -15,6 +15,10 @@
 #    pragma warning(disable : 4100)
 #endif
 
+// PY_SSIZE_T_CLEAN is required for all # variants of formats (s#, y#, etc.).
+// See https://docs.python.org/3/c-api/arg.html
+#define PY_SSIZE_T_CLEAN
```

Comment:
Moved here from ValueFactoryManager.h, this must be included before `Python.h`. This seems like a better place.